docs/central-upgrade.rst

```diff
@@ -385,6 +385,12 @@ This is *critical infrastructure upgrade*. In particular, it upgrades the includ
    
        #. **Upgrade your database server**. We recommend using the latest point release of PostgreSQL 14 that is available.
 
+       #. **Regenerate optimizer statistics**. You need to regenerate all database statistics to avoid performance issues.
+   
+          .. code-block:: sql
+   
+            ANALYZE VERBOSE;
```

Comment:
Should make clear that this is in the db. Folks are going to run this at the shell.